### PR TITLE
fix(devops): Specify canister IDs

### PR DESCRIPTION
**Pull request by bitdivine**
# Motivation
The signer uses the cycles ledger for payment.  For this to work with pre-built Wasms, the cycles ledger needs to be deployed to the correct canister ID.

# Changes
- Fill in the canister dependencies:
  - the backend depends on the signer and the cycles ledger
  - the signer depends on the cycles ledger
- Specify which canister ID the signer and cycles ledger should be deployed to, so that pre-built Wasms will interact correctly.

# Tests

### CODE REVIEW

File: scripts/generate.sh [hunk 1]
```diff
@@ -32,6 +32,7 @@ function install_did_files() {
       canister_name="${array[0]}"
       source="${array[1]}"
       filename="${source##*/}"
+      filename="${filename//-/_}"
```
Comment (bitdivine) [inline scripts/generate.sh:35]: The cycles-ledger has a hyphen in the did file name.  dfx uses underscores instead.

Comment (lmuntaner) [reply scripts/generate.sh:35]: Can you add this comment in the code?

**Review by lmuntaner — APPROVED** [review]
LGTM! Just a minor comment.

File: scripts/generate.sh (hunk 1 above)
Comment (bitdivine) [reply scripts/generate.sh:35]: Done

**Pull request merged**